src/main/java/com/google/sps/TripDay.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+
+/**
+ * TripDay objects store the trip/route information for a single day.
+ */
+public class TripDay {
+  private String origin;
+  private String destination;
+  private ArrayList<String> locations;
```

Comment:
Two questions:
1. Will these variables always store Strings, or are you going to later use the Google Maps Java Client to store place objects here? (I just saw in the `TripDayTest` class that these Strings are seemingly **place IDs** -- if this is the case, can you add a comment somewhere explaining this?)
2. Say the last location (POI) is also where the user wants to end their day. Is it included in both `locations` and as the `destination`, or is it no longer included at the end of `locations`?

Reply:
Expanding on this; on the frontend, I haven't provided users any options for putting a `destination` different than their `origin`, because at least for the MVP I assumed this would be the same. Was this a fair assumption, or should I include a spot for where they end their day as well?

My main concern here is that this could become overly complicated once we start to add multiple days -- I'd have to dynamically allow adding different `origin` and `destination` values for every day.

Reply:
1. I have these as Strings (for Place IDs) for now, but this could change depending on how I end up using the Java Client! Still exploring/learning about the Client so will keep you posted on that. Have added a comment explaining that these params are Place ID Strings for now!

2. If the last POI is the last stop on the route, then that will be stored only in destination, not locations. For the MVP, I am just assuming that the origin and destination are the same, so this should not be an issue.

For the MVP, I think the form is fine with just having the user input an origin because we are assuming that all trips will be round trips to and from that location, so I can just set the destination to the origin when calling the DirectionsQuery. Beyond the MVP, we can talk about how we want it to look, but I don't think there is any immediate action required for now.